addOns/help/src/main/javahelp/contents/releases/2.12.0.html

```diff
@@ -73,6 +149,7 @@ <H2>Add-Ons</H2>
 <H3>New Add-Ons</H3>
 The following add-ons are included by default in this release for the first time:
 <ul>
+  <li><a href="https://www.zaproxy.org/docs/desktop/addons/database/">Database</a> - provides the SQLite database engine.</li>
```

Comment:
Should we be less specific here? I know sqlite is currently provided but I think the intention was to provide others in the future.